services/self-service/src/main/java/com/epam/dlab/backendapi/service/impl/ProjectServiceImpl.java

```diff
@@ -344,7 +344,8 @@ private String getUpdateBudgetAudit(ProjectDTO p) {
 		Integer value = Optional.ofNullable(get(p.getName()).getBudget())
 				.map(BudgetDTO::getValue)
 				.orElse(null);
-		return String.format(AUDIT_UPDATE_BUDGET, value, p.getBudget().getValue());
+		boolean monthlyBudget = get(p.getName()).getBudget().isMonthlyBudget();
```

Comment:
We have already made a call to Mongo in line 344. Please refactor code do avoid a double call to DB.

Reply:
Code base has been adjusted